src/GameSelector/index.tsx

```diff
@@ -0,0 +1,68 @@
+import { Button, Center, FormControl, Input, VStack } from "native-base";
+import React from "react";
+
+export interface GameSelectorFormData {
+  gameId: number | undefined;
+}
+
+const GameSelectorForm = () => {
+  const [formData, setData] = React.useState<GameSelectorFormData>({
+    gameId: undefined,
+  });
+  const [errors, setErrors] = React.useState({});
```

Comment:
Do you already need an error object?

Reply:
I use it in my form to tell the user if he hasn't filled an id

Reply:
My idea behind this question is that as you only have one error raised on this form, you could have simplified the task by using a boolean like `hasError` instead of an object. So you don't need the test anymore `'gameId' in errors ?`